xml/ns-System.Drawing.xml

```diff
@@ -1,6 +1,8 @@
 <Namespace Name="System.Drawing">
   <Docs>
-    <summary>Provides access to GDI+ basic graphics functionality. More advanced functionality is provided in the <see cref="N:System.Drawing.Drawing2D" />, <see cref="N:System.Drawing.Imaging" />, and <see cref="N:System.Drawing.Text" /> namespaces.</summary>
+    <summary>Provides access to GDI+ basic graphics functionality. More advanced functionality is provided in the <see cref="N:System.Drawing.Drawing2D" />, <see cref="N:System.Drawing.Imaging" />, and <see cref="N:System.Drawing.Text" /> namespaces.
+    System.Drawing is not supported in ASP.NET, ASP.NET Core, or Windows Services. We recommend ImageSharp for these platforms. For more information, see <see href="https://github.com/SixLabors/ImageSharp">SixLabors.ImageSharp</see>.
+    </summary>
```

Comment:
AFAIK, System.Drawing is supported on ASP.NET, ASP.NET Core and Windows Services. You'll need the System.Drawing.Common package reference inn ASP.NET Core. I'm not sure what makes Windows Services special.

@davidfowl was just pointing out that there are more modern alternatives like ImageSharp that don't use GDI+. This is what @shanselman has to say about System.Drawing vs ImageSharp [his blogpost](https://www.hanselman.com/blog/how-do-you-use-systemdrawing-in-net-core):

> There's lots of great options for image processing on .NET Core now! It's important to understand that this System.Drawing layer is great for existing System.Drawing code, but you probably shouldn't write NEW image management code with it. Instead, consider one of the great other open source options.
>
> - ImageSharp - A cross-platform library for the processing of image files; written in C#
>
>   - Compared to `System.Drawing` ImageSharp has been able to develop something much more flexible, easier to code against, and much, much less prone to memory leaks. Gone are system-wide process-locks; ImageSharp images are thread-safe and fully supported in web environments.

Reply:
After reading the warning at the bottom of the page, I now understand what is meant by System.Drawing not being supported on ASP.NET, ASP.NET Core and Windows Services.

> Classes within the System.Drawing.Imaging namespace are not supported for use within a Windows or ASP.NET service. Attempting to use these classes from within one of these application types may produce unexpected problems, such as diminished service performance and run-time exceptions.

I think we should move this up to the top of the docs because it clearly explains why it's not supported.